### ISSUE TITLE
State is not reset after form completion

### ISSUE THREAD

**Issue by RhysLees** [AbanoubNassem/filament-grecaptcha-field#11]
After submitting the form, you cannot submit again as the field does not reset

https://github.com/AbanoubNassem/filament-grecaptcha-field/assets/43909932/63b0358c-b866-4948-a441-6e9f8470eb47

```php
GRecaptcha::make('captcha')
    ->label('')
    ->validationMessages([
        'required' => 'You must complete the captcha.',
    ]),
```

**Comment by AbanoubNassem** [comment]
check the console does it output any error messages ?

**Comment by ZayRTun** [comment]
Yes, I am having this same issue, where the it does not reset. And there is no error in the console too.

**Comment by Umar-Farooq-Shafi** [comment]
any update?

**Comment by Sauruz** [comment]
Same problem here. When there is an error in a field, the recaptcha does not reset. Is seems the resetCaptcha event in g-recaptcha.blade.php doesnt get fired when Livewire is reloaded.

**Comment by ZayRTun** [comment]
Dispatch an event from the livewire component
```
$this->dispatch('reset-recaptcha');
```

And in blade
```
x-on:reset-recaptcha="window.grecaptcha.reset()"
```
Hope that helps...

**Comment by AbanoubNassem** [comment]
I have no clue what `filament` introduced to break the reset event, am looking into it

**Comment by alisilarbi** [comment]
@AbanoubNassem I have issues where the captcha doesn't get dispalyed on the last step of my wizard. as if the form "changed"

**Comment by AbanoubNassem** [comment]
closes #11  with release v0.0.8
